averages.js

```diff
@@ -0,0 +1,54 @@
+function total(nums){
+  numTotal = nums.reduce( function(sum, num){
```

Comment:
Make sure to use var, let, or const keywords to ensure you don't declare any global variables 😊